README.md

```diff
@@ -31,13 +31,13 @@ Use the following configuration to connect to the currently deployed test setup.
 ```javascript
 import { TLNetwork } from 'trustlines-clientlib'
 
-const config = {
+const TLNetworkConfig = {
```

Comment:
Why did you rename this? variable names should start with lower case letter

Reply:
I did not find config to be explicit enough in most parts of the code and renamed it to TLNetworkConfig, I can understand that it does not matter in this particular place actually.
I can rename all of them to tlNetworkConfig if you want or back to config.

Reply:
i am fine with tlNetworkConfig